### PR TITLE
[GORA-639] Updated YCSB dependency and added developer details to root pom

### PR DESCRIPTION
**Pull request by sneceesay77**
YCSB dependency is now available in maven central, I have now updated the pom file and the code accordingly.

### CODE REVIEW

**Comment by lewismc** [comment]
@sneceesay77 please resolve conflict in pom.xml thank you

**Comment by lewismc** [comment]
Also, @sneceesay77 please always open a [JIRA ticket](https://issues.apache.org/jira/browse/GORA) that way we can include you contribution in the next release report thank you.

File: pom.xml [hunk 1]
```diff
@@ -859,7 +868,7 @@
     <test.container.version>1.4.2</test.container.version>
     
     <!-- gora-benchmark and version dependencies -->
-    <ycsb.version>0.17.0-SNAPSHOT</ycsb.version>
+    <site.ycsb.version>0.17.0</site.ycsb.version>
```
Comment (djkevincr) [inline pom.xml]: Please enable gora-benchmark from parent pom.
```
<!-- <module>gora-benchmark</module> -->
```

Comment (sneceesay77) [reply pom.xml]: Done

**Comment by djkevincr** [comment]
@sneceesay77 Please go ahead and merge PR, let see whether you have any issues with commit rights to the repo ;)

**Pull request merged**